app/views/inbox/_inbox_comment_contents.html.erb

```diff
@@ -26,7 +26,6 @@
   <% end %>
 </div>
 
-<% # This feedback_comment used to be inbox_comment... not sure why %>
 <blockquote class="userstuff">
-  <%= raw sanitize_field(feedback_comment, :comment_content) %>
+  <%= raw sanitize_field(feedback_comment, :comment_content, strip_images: feedback_comment.ultimate_parent.is_a?(AdminPost)) %>
```

Comment:
Should there be a test for this, either in [comments_adminposts.feature](https://github.com/otwcode/otwarchive/blob/master/features/comments_and_kudos/comments_adminposts.feature) or [inbox.feature](https://github.com/otwcode/otwarchive/blob/master/features/comments_and_kudos/inbox.feature)?